qa/workunits/rbd/cli_migration.sh

```diff
@@ -102,23 +135,32 @@ test_import_native_format() {
     local base_image=$1
     local dest_image=$2
 
-    rbd migration prepare --import-only "rbd/${base_image}@2" ${dest_image}
-    rbd migration abort ${dest_image}
+    # if image is from namespace
+    local namespace=""
+    if [[ "$base_image" == rbd/*/* ]]; then
+        namespace=$(basename "$(dirname "$base_image")")
+    fi
+
+    # Execute only if it's a non-namespace based image
+    if [[ -z "$namespace" ]]; then
+        rbd migration prepare --import-only "rbd/${base_image}@2" ${dest_image}
+        rbd migration abort ${dest_image}
+    fi
 
     local pool_id=$(ceph osd pool ls detail --format xml | xmlstarlet sel -t -v "//pools/pool[pool_name='rbd']/pool_id")
     cat > ${TEMPDIR}/spec.json <<EOF
 {
   "type": "native",
   "pool_id": ${pool_id},
-  "pool_namespace": "",
-  "image_name": "${base_image}",
+  "pool_namespace": "${namespace}",
+  "image_name": "$(basename "$base_image")",
```

Comment:
I'd suggest renaming the existing `base_image` and `dst_image` variables to `base_image_spec` and `dst_image_spec` respectively, the new `namespace` to `base_namespace` and having `base_image` contain just the image name so that `basename` doesn't need to be called repeatedly.

Reply:
Done as per the suggestion.